src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/custom.py

```diff
@@ -143,44 +143,50 @@ def enable_local_git(resource_group, name, slot=None):
 
     return {'url' : _get_git_url(client, resource_group, name, slot)}
 
-def create_app_service_plan(resource_group, name, tier=None, number_of_workers=None,
+def create_app_service_plan(resource_group, name, sku, number_of_workers=None,
                             location=None):
     client = web_client_factory()
+    sku = _normalize_sku(sku)
     if location is None:
         location = _get_location_from_resource_group(resource_group)
 
-    sku_name = _get_sku_name(tier)
     #the api is odd on parameter naming, have to live with it for now
-    sku = SkuDescription(name=tier, tier=sku_name, capacity=number_of_workers)
-    plan_def = ServerFarmWithRichSku(location, server_farm_with_rich_sku_name=name, sku=sku)
-    #TODO: handle bad error on creating too many F1 plans:
```

Comment:
It will be tracked as a bug per mail thread yesterday, hence no need for comment